iipsrv/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3
+
+RUN apk update
+RUN apk upgrade
+RUN apk add --no-cache \
+  autoconf \
+  automake \
+  build-base \
+  jpeg-dev \
+  libtool \
+  m4 \
+  make \
+  pkgconfig \
+  tiff-dev
+
+RUN wget https://github.com/ruven/iipsrv/archive/refs/tags/iipsrv-1.1.zip
+RUN unzip iipsrv-1.1.zip
+WORKDIR iipsrv-iipsrv-1.1
+RUN ./autogen.sh
+RUN ./configure
+RUN make
+
+ENV JPEG_QUALITY 100
+ENV MAX_LAYERS 4
+ENV MAX_CVT 7000
```

Comment:
I would suggest:

`JPEG_QUALITY 90`

and remove `MAX_LAYERS` and `MAX_CVT`. This is because these are parameters that restrict access to the images -- MAX_LAYERS restricts the number of layers that can be addressed in a JPEG2000 file, and MAX_CVT restricts the size of the image in the largest dimension. These parameters are useful if you're delivering images as a public service, to help prevent server overload or people downloading full-size, full-resolution images. But for an internal service like Rodan they're probably not necessary.

Setting JPEG_QUALITY lower is because 90 is the optimal size / quality parameter for first-generation derivatives. That is, if you go over 90, you're not gaining noticable visual quality, you're only increasing the size of the resulting image. It would be significant if you were creating multiple generations of derivatives, but because you're (presumably) coming from the full-resolution TIFF or JPEG2000 all the time, you don't need that.

Reply:
Done. Thanks for the comments.